src/data_generators.py

```diff
@@ -35,6 +34,10 @@ def D_data_generator(
         attribute = 'disease'  # name of attribute in npz archive
         age_max = 87.72895277  # used for scaling age values
         n_condns = 3  # number of conditions
+    else:
+        attribute = 'age'  # name of attribute in npz archive
+        age_max = 100
+        n_condns = 1
```

Comment:
For custom datasets, `attribute` and `age_max` should likely be passed through `train_script.py`'s CLI

Also, this ignores the user-specified `n_condns` in the new CLI for custom data.